lib/utils/dom.js

```diff
@@ -63,12 +63,6 @@ click.isProgrammaticEvent = e => {
 	return e.clientX === 1 && e.clientY === 1;
 };
 
-export function mousedown(obj, button = 0) {
```

Comment:
Is this still used for clicking native reddit elements, like keyboard nav invoking upvote/downvote?